conmon/conmon.c

```diff
@@ -152,11 +175,11 @@ int main(int argc, char *argv[])
 	/* Create the container */
 	if (terminal) {
 		snprintf(cmd, CMD_SIZE,
-			 "%s create %s --pid-file pidfile --console %s",
-			 runtime_path, cid, slname);
+			 "%s create %s %s --pid-file %s --console %s",
```

Comment:
@runcom current runc works with this PR because oci.go sets [`cmd.Dir` to the bundle path](https://github.com/kubernetes-incubator/cri-o/blob/master/oci/oci.go#L110) and thus runc works directly under the bundle path cwd. Runc's `--bundle` is optional and only does a `Chdir()` when it's set. So my PR here passes a second argument to runc that is ignored (Which is another spec violation).